contributor-ladder.md

```diff
@@ -0,0 +1,157 @@
+    * Participating in community discussions
+    * Helping other users
+    * Submitting bug reports
+    * Labeling issues
+    * Triaging bugs
+    * Testing releases
+    * Participating in community calls
+
+### Contributor
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include: 
+    * Must follow the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+    * Following the project contributing guide
+* Requirements (one or several of the below):
+    * Occasionally submits PRs
+    * Submits doc fixes
+    * Regularly shows up at meetings, takes notes
+    * Answers questions from other community members
+    * Submits feedback on issues and PRs
+    * Runs a meetup or other event
+    * [TODO: other small contributions]
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+
+### Organization Member
+<!--This role is used by many projects where you have to be a regular contributor to have the right to vote in project elections, or to be able to operate project
+CI/CD automation.-->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: An Organization Member is an established contributor who regularly participates the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Organization Member must meet the responsibilites and has the requirements of a Contributor, plus:
+
+* Responsibilities include: 
+    * Continues to contribute regularly, as demonstrated by having at least [TODO: Number] [TODO: Metric] a year, as demonstrated by [TODO: contributor metrics source].
+<!-- Example: "as demonstrated by having at least 50 Github contributions per year, as shown by Devstats"-->
+
+* Requirements:
+    * Must have successful contribitons to the project, including at least one of the following:
+        * [TODO: Number] accepted PRs,
+        * Reviewed [TODO: Number] PRs,
+        * Resolved and closed [TODO: Number] Issues,
+        * Become responsible for a key project management area,
+        * Or some equivalent combination or contribution
+    * Must have been contributing for at least [TODO: Number] months
+    * Must be actively contributing to at least one project area
+    * Must have two sponsors who are also Organization Members, at least one of whom does not work for the same employer
+    * [TODO: other requirements] <!--TODO: such as repository access or enabling 2FA on their Github account -->
+
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
```

Comment:
Are we to assume that ALL projects will be under ci-bot control?

Reply:
That is still under investigation which depends on the CI/CD community needs/decision.